dali/python/nvidia/dali/pipeline.py

```diff
@@ -169,13 +169,45 @@ def epoch_size(self, name = None):
         ----------
         name : str, optional, default = None
             The reader which should be used to obtain epoch size.
+        with_padding : bool, optional, default = True
+            If is set to true the data set size is rounded to multiply of
```

Comment:
```suggestion
            If is set to true the data set size is rounded to multiple of
```

Reply:
You didn't handle the `with_padding` argument at all.

Reply:
It is no longer needed as we have meta and this was added in this PR. Removed.